x/crosschain/keeper/msg_server_vote_outbound.go

```diff
@@ -183,11 +272,14 @@ func (k Keeper) FundGasStabilityPoolFromRemainingFees(
 	return nil
 }
 
-// percentOf returns the percentage of a number
-func percentOf(n *big.Int, percent int64) *big.Int {
-	n = n.Mul(n, big.NewInt(percent))
-	n = n.Div(n, big.NewInt(100))
-	return n
+// PercentOf returns the percentage of a number
+func PercentOf(n math.Uint, percent uint64) math.Uint {
+	percentUint := math.NewUint(percent)
+	result := n.Mul(percentUint)
+	hundred := math.NewUint(100)
+	result = result.Quo(hundred)
+
+	return result
```

Comment:
also can be a bit more compressed eg: return result.Quo(hundred) etc, but up to you

maybe we can not export this one or have it in some util